app/docker-compose.yml

```diff
@@ -30,6 +30,7 @@ services:
     environment:
       - "discovery.type=single-node"
       - "xpack.security.enabled=false"
+      - "ES_JAVA_OPTS=-Xms512m -Xmx512m"
```

Comment:
FYI - I think we would need more memory for the JVM (that is the ES server) as do more interesting stuff on the ES (like running aggregations, searches, etc.). The settings you have here may be a good start (although I dont see why the Xms (starting memory) and Xmx (max memory) should be the same)

Reply:
oh definitely, to be clear these are settings for docker-compose. Agree that we'd give a lot more memory to the ES server on AWS when I spin that up